### ISSUE TITLE
ChefDK app manifest

### ISSUE THREAD

**Issue by Hazardius** [lukesampson/scoop#2113]
**tl;dr:** There's no app manifest for ChefDK and I think it's a good idea to add one to bucket here or to extras.

Recently I started to learn Chef so I've tried to install it on my Windows machine(of course - by scoop :yum: ). But I couldn't find it in any bucket I'm following. Then I tried to add it myself, but there's two problems: I'm completely clueless on PS scripts and as a beginner in Chef I'm not fully aware of which binaries should I put into the app manifest or if any at all. Adding ChefDK to the scoop is a good idea as Chef feels like becoming a standard amongst the DevOps.

Here are some of the informations I gathered while trying to do it's app manifest for test bucket earlier:
- [Chef .msi files are here.](https://downloads.chef.io/chefdk/2.5.3#windows)
- [Chef homepage.](https://www.chef.io/)
- [It's license is here.](https://downloads.chef.io/chefdk/stable/2.5.3/windows/2016/license)

**Comment by r15ch13** [comment]
- What about the other tools listed here https://downloads.chef.io/?
- The ChefDK provides a bunch of executables, which are important for shiming or should they be added to `PATH`?
- Which `bin` directory? `bin` or `embedded\bin`?

**Comment by brandon93s** [comment]
If you're using Chef, you're going to need the kitchen sink (pun intended). I suggest adding them to `PATH`.

**Comment by rasa** [comment]
This issue belongs in the [Main](https://github.com/ScoopInstaller/Main) bucket. Unfortunately, we cannot move this issue on this end, so it is being closed.
If this is still an issue, please create a new issue in the Main bucket via https://github.com/ScoopInstaller/Main/issues/new?body=Moved+from+https%3A%2F%2Fgithub.com%2Flukesampson%2Fscoop%2Fissues%2F2113
